sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/storage/fileshare/AzureStorageFileShareAutoConfiguration.java

```diff
@@ -28,14 +28,13 @@
 @ConditionalOnAnyProperty(prefix = "spring.cloud.azure.storage.fileshare", name = { "account-name", "endpoint", "connection-string" })
 public class AzureStorageFileShareAutoConfiguration extends AzureServiceConfigurationBase {
 
-    public AzureStorageFileShareAutoConfiguration(AzureGlobalProperties azureGlobalProperties) {
-        super(azureGlobalProperties);
+    public AzureStorageFileShareAutoConfiguration() {
     }
 
     @Bean
     @ConfigurationProperties(AzureStorageFileShareProperties.PREFIX)
-    public AzureStorageFileShareProperties azureStorageFileShareProperties() {
-        return loadProperties(this.azureGlobalProperties, new AzureStorageFileShareProperties());
+    public AzureStorageFileShareProperties azureStorageFileShareProperties(AzureGlobalProperties azureGlobalProperties) {
+        return loadProperties(azureGlobalProperties, new AzureStorageFileShareProperties());
     }
```

Comment:
Same here.

Reply:
I will test the code again  to check the logic here.